tasks/bal_getPoolAddress.ts

```diff
@@ -2,6 +2,7 @@ import { getPoolAddress } from '@balancer-labs/balancer-js';
 import IVaultArtifact from '@balancer-labs/v2-deployments/dist/tasks/20210418-vault/abi/Vault.json';
 import { task } from 'hardhat/config';
 import type { HardhatRuntimeEnvironment } from 'hardhat/types';
+import weightedPool_abi from '../node_modules/@balancer-labs/v2-deployments/dist/tasks/20210418-weighted-pool/abi/WeightedPool.json';
```

Comment:
`import weightedPool_abi from '@balancer-labs/v2-deployments/dist/tasks/20210418-weighted-pool/abi/WeightedPool.json';`

Reply:
ok, I've updated it.